ga4gh/cli.py

```diff
@@ -401,10 +434,10 @@ def _textOutput(self, gaObjects):
             # TODO add in some more useful output here.
             print(read.id)
 
-
 # ListReferenceBases is an oddball, and doesn't fit either get or
 # search patterns.
 
+
```

Comment:
the commit diffs should be as small as possible... try to avoid changing lines unnecessarily